src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,113 @@
 package core.basesyntax;
 
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double GROWTH_INDEX = 1.5;
```

Comment:
this is unnecessary constant because you have `int newCapacity = Math.max(elements.length + (elements.length >> 1), minCapacity);` inside `ensureCapacity` method. Your new capacity always will be greater in 1.5 times.